fmf/utils.py

```diff
@@ -853,3 +853,26 @@ class JsonSchemaValidationResult(NamedTuple):
 
     result: bool
     errors: List[Any]
+
+
+class PatternRepl(NamedTuple):
+    pattern: str
+    repl: str
+
+
+def BETTER_NAME(source):
+    """ Splits sed-like input into pattern/repl pair suitable for re.sub """
+    try:
+        if source[0] != source[-1] != '/':
```

Comment:
Why not support any character and use `string.split` to avoid processing any escape characters? Than everything would be escaped/processed as normal regex patterns. There might be some trickery needed to escape `source[0]`, but that should have a predictable pattern to find-and-replace with normal string operations before parsing regex: `'\' + escape_char` -> `escape_char`

This would be important to support because escaping `/very/long/path/to/my/data` will be a nightmare to read.

Reply:
If I don't need to implement escaping, as in: Set the leading character properly as it cannot be used in pattern nor in repl then +1. I didn't think it would be acceptable ;)

Reply:
But even with escaping it should be easy to change.